backend-worker/worker.py

```diff
@@ -427,26 +432,32 @@ def pixelpercents_under_parcel(parcel_wkt_epsg3857, source_raster_path):
     return_values = {}
     n_values_under_parcel = numpy.sum(counts)
     for lulc_code, pixel_count in zip(values_under_parcel, counts):
-        return_values[lulc_code] = round(
+        # cast lulc_codes to int from numpy_int16 for future json dump call
+        # which does not allow numpy types for keys
+        return_values[int(lulc_code)] = round(
             pixel_count / n_values_under_parcel, 4)
 
     return return_values
 
 
 def do_work(ip, port):
     local_appdata_dir = 'appdata'
-    job_queue_url = f'{ip}:{port}/jobsqueue/'
+    job_queue_url = f'http://{ip}:{port}/jobsqueue/'
```

Comment:
Oh yeah, this protocol is good to include.  If we're adding this here, could we also assert that the user has provided an IPv4 address?  On the other hand, if the user does provide a protocol in addition to the ipv4 address, this will fail loudly so maybe this isn't important.